.github/actions/pnpm/action.yml

```diff
@@ -5,7 +5,7 @@ runs:
   steps:
     - uses: pnpm/action-setup@v2.4.0
       with:
-        version: 7.32.5
+        version: 8.6.11
```

Comment:
I don't have an opinion here, but was curious what you thought - would it make sense for CI to do `version: 8` instead, so that it always uses the latest version of `pnpm`?  Do we care if CI is on a different version than what we may be running locally?  We were previously on "use latest" like this and caught a bug with `pnpm` that got resolved rather quickly.

There could be some downsides to this though.  I don't know if it's totally worth it, but thought I'd get your opinion.  I can't remember if renovate/dependabot will auto-upgrade this version to match what is in the volta / engines config.

Reply:
No opinion here either 🤷‍♂️